packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/amplify-table-manager-handler.ts

```diff
@@ -130,6 +130,27 @@ const processOnEvent = async (event: AWSCDKAsyncCustomResource.OnEventRequest):
   let result;
   switch (event.RequestType) {
     case 'Create':
+      if (tableDef.isImported) {
+        // TODO: Add import validation
```

Comment:
What validation needs to be added?

Reply:
The validation is to ensure the DDB properties align (Example: KeySchema, SSESpecification, etc.). WIP PR: https://github.com/aws-amplify/amplify-category-api/pull/2696